src/swsssdk/configdb.py

```diff
@@ -17,11 +17,12 @@
     config_db.listen()
 
 """
-
 import sys
 import time
 from .dbconnector import SonicV2Connector
 
+PY3K = sys.version_info >= (3, 0)
```

Comment:
>K [](start = 3, length = 1)

What does K mean?

Reply:
py3k is py3000, py3.0 same meaning.

Reply:
It appears "Python 3000" (or "py3k") was the initial name for what was eventually changed to "Python 3.0" to keep in line with the old versioning scheme before its actual release. I wasn't aware of the early name, and others might not be, either. We can always change it to "PY3X" or similar in the future.

Whatever we choose, we should probably use the same convention in all of our scripts whenever we need to make them compatible with both Python 2 and Python 3.